dbOperations/hooks/useWaitlist.js

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from "@tanstack/react-query";
+import { collection, getDocs } from "firebase/firestore";
+import { useAuthContext } from "~/context/Auth";
+import { db } from "~/firebaseConfig";
+
+export const useWaitlist = ({ enabled = true } = {}) => {
+  const { currentTeamId } = useAuthContext();
+  const { data, error, isLoading } = useQuery({
+    queryKey: ["waitlist"],
+    queryFn: async () => {
+      console.log("fetching waitlist");
+      const newWaitlist = {};
+      // Fetch all time records
```

Comment:
looks like a stray comment from a ctrl C + ctrl V lol
```suggestion
      // Fetch all Entries on the Waitlist
```